### PR TITLE
Edit/pre release docs cleanup

### PR DESCRIPTION
**Pull request by Erin-Cecele**
Thank you for contributing to the Quire Documentation & Website! Please complete the form below to submit your pull request for review.

*For the Title of this pull request, please use the format "Type/Issue-#: Brief description." For Type, the options are Edit, Add, Translate. Issue-# is only needed if this pull request addresses an existing issue.*

### Checklist 

Please put an X within the brackets that apply `[X]`.

- [X] I have read the [CONTRIBUTING.md](https://github.com/thegetty/quire/blob/main/CONTRIBUTING.md) file.

- [X] I have made my changes in a new branch and not directly in the main branch

- [ ] I am requesting feedback on a draft pull request

### Is this pull request related to an open issue? If so, what is the issue number?

No

### Please describe the goal of this pull request and the changes that were made.

These are refinements to sections that have already had major PRs merged. It's a mishmash of the things I felt needed to be tightened up/clarified before launch.

### CODE REVIEW

File: content/docs-v1/v1-tutorial.md [hunk 1]
```diff
@@ -45,6 +45,7 @@ In addition to telling Quire what to do, the command-line shell can also be used
 3. Type `cd` followed by the name of a directory you want to navigate to. For example, `cd Downloads` will take you into the Downloads directory.
 4. Run `ls` again to view the contents of that folder.
 5. Pull up your Finder or File Explorer to compare the listed files.
+6. Type `cd` again to go back to your home directory.
```
Comment (geealbers) [inline content/docs-v1/v1-tutorial.md:48]: on WIndows, it **has** to be `cd ~` to go to the home directory. `cd ~` also works on Mac so maybe that should be used here.

File: content/docs-v1/v1-tutorial.md [hunk 2]
```diff
@@ -246,7 +250,8 @@ Once you've played around with adding content and customizing your project, it's
 
 1. In your Terminal, stop the preview process by pressing Control-C.
 2. To create the various formats of your publication type `quire build` and press enter.
-3. In your Finder, navigate to the `X` directory and open the PDF to view it.
+3. Use your Finder or File Explorer to navigate to your project directory.
+4. Navigate to the `public` directory and open the file marked `pdf.html` to view a PDF of your project.
```
Comment (geealbers) [inline content/docs-v1/v1-tutorial.md:254]: `pdf.html` isn't the pdf itself, but rather an HTML file that is used to generate the PDF. I actually don't know where the PDF will ultimately show up

File: content/docs-v1/getting-started.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ aliases:
   - /documentation/getting-started/
 ---
 
-Before getting started, make sure you have [installed Quire](/docs-v1/install-uninstall/) and downloaded a freely available {{< q-def "text editor" >}}. We also recommend reviewing the [*Quire Basics Tutorial*](/learn/tutorial/), a step-by-step introduction to the fundamentals of Quire.
+Before getting started, make sure you have [installed Quire](/docs-v1/install-uninstall/) and downloaded a freely available {{< q-def "text editor" >}}. We also recommend reviewing the [*Quire Basics Tutorial*](/docs-v1/v1-tutorial/), a step-by-step introduction to the fundamentals of Quire.
```
Comment (geealbers) [inline content/docs-v1/getting-started.md:10]: should the filename for the tutorial just be `tutorial.md` since it's already in the `/docs-v1/` directory?

**Comment by Erin-Cecele** [comment]
I will finish renaming Learn and Tutorial in a separate branch/PR. My PRs have started to overlap and it's getting messy. Want to start a new branch with the most up-to-date docs-11ty branch.

**Pull request merged**